libraries/botframework-connector/src/auth/managedIdentityAppCredentials.ts

```diff
@@ -0,0 +1,53 @@
+/**
+ * @module botframework-connector
+ */
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { TokenResponse } from 'adal-node';
+import { AppCredentials } from './appCredentials';
+import { JwtTokenProviderFactoryInterface } from './jwtTokenProviderFactoryInterface';
+import { ManagedIdentityAuthenticator } from './managedIdentityAuthenticator';
+import { ok } from 'assert';
+
+/**
+ * Managed Service Identity auth implementation.
+ */
+export class ManagedIdentityAppCredentials extends AppCredentials {

```

Comment:
The access level for this class doesn't match [.NET](https://github.com/microsoft/botbuilder-dotnet/pull/5829/files#diff-5b38e347048b007edd6393cc510e83ce6fc883d48c38b1ff6262edc62a21985eR14); to do so, the class needs to be exported from `src/auth/index.ts`.